site/guide/3-overview.md

```diff
@@ -214,9 +215,9 @@ You can find async-ready libraries on [crates.io](https://crates.io) with the
 
 ! note
 
-  Rocket 0.5 uses the tokio (0.2) runtime. `Rocket::launch()` will automatically
-  start a runtime for you, or you can create a runtime yourself and use
-  `Rocket::spawn()`.
+  Rocket 0.5 uses the tokio (0.2) runtime. The runtime is started for you
+  if you use `#[rocket::main]`, but you can still `launch()` a rocket
+  instance on a custom-built `Runtime`.
```

Comment:
This requires some pointers on how to do so. Is that material available?

Reply:
We should perhaps open up an issue, if one doesn't already exist, to document all of the things that need to be documented before we release `async`.